src/components/PupilViews/PupilIntro/PupilIntro.view.tsx

```diff
@@ -81,109 +85,118 @@ export const PupilViewsIntro = (props: PupilViewsIntroProps) => {
       topNavSlot={topNavSlot}
       bottomNavSlot={bottomNavSlot}
     >
-      <OakFlex $width={"100%"} $alignItems={"center"} $flexDirection={"column"}>
-        <OakFlex
-          $flexDirection={"column"}
-          $gap={["space-between-m", "space-between-l", "space-between-xl"]}
-          $maxWidth={["100%", "all-spacing-22", "all-spacing-23"]}
-          $minWidth={["100%", "all-spacing-21", "all-spacing-23"]}
-          $ph={["inner-padding-m", "inner-padding-none"]}
+      <OakGrid
+        $cg="all-spacing-4"
+        $maxWidth={["100%", "all-spacing-22", "100%"]}
+        $mb={["space-between-none", "space-between-s"]}
+        $mh="auto"
+        $ph={["inner-padding-m", "inner-padding-xl", "inner-padding-none"]}
+        $minHeight="100%"
+        $gridTemplateRows={[
+          "min-content min-content 1fr",
+          "min-content min-content 1fr",
+          "min-content 1fr min-content",
+        ]}
```

Comment:
This ensures that the licence sticks to the bottom of the page while allowing the page to scroll if needed